docs/_tutorials/scenarios/scenarios.rst

```diff
@@ -310,6 +310,60 @@ supports this setting.
     # Not implemented yet
 
 
+Incremental Semantic Segmentation
+---------------------------------
+
+Brought by [Michieli et al. ICCV-W 2017](https://openaccess.thecvf.com/content_ICCVW_2019/papers/TASK-CV/Michieli_Incremental_Learning_Techniques_for_Semantic_Segmentation_ICCVW_2019_paper.pdf)
+and [Cermelli et al. CVPR 2020](https://arxiv.org/abs/2002.00718), continual learning
+for segmentation is very different from previous scenarios.
+
+Semantic segmentation aims at classifying all pixels in an image, therefore
+multiple classes can co-exist in the same image. This distinction leads to three kinds of scenarios:
+
+- **Sequential**: where for a given task T, with current classes C, the model sees all
+  images that contain at least one pixel labeled as a current classes C. If the image contains
+  future classes, yet unseen, then it is discarded. In the sequential setting, all pixels
+  are labeled, either with a old or current class label, background label (0), or
+  unknown label (255).
```

Comment:
c'est pas un peu cheaté la label "je sais pas encore ce que c'est mais dans le futur on me dira" ?

Reply:
Yes Sequential, and Disjoint aren't realistic. But they exists, not my fault ;)
Overlap ftw!